code/game/objects/items/weapons/storage/bible.dm

```diff
@@ -27,6 +37,68 @@
 	new /obj/item/stack/spacecash(src)
 	new /obj/item/stack/spacecash(src)
 	new /obj/item/stack/spacecash(src)
+
+
+/obj/item/storage/bible/attack_self(mob/living/carbon/human/H)
+	if(!istype(H))
+		return
+	if(ticker && !ticker.Bible_icon_state && H.mind.assigned_role == "Chaplain")
+		//Open bible selection
+		var/dat = "<html><head><title>Pick Bible Style</title></head><body><center><h2>Pick a bible style</h2></center><table>"
+
+		var/i
+		for(i = 1, i < biblestates.len, i++)

```

Comment:
`for(var/i in 1 to GLOB.biblestates.len)`

Reply:
OH GAH GLOBS...I FORGOT...